src/pages/guidelines/icons/contribute.mdx

```diff
@@ -39,8 +39,8 @@ guidelines to ensure visual consistency and proper formatting.
 
 ## Making an icon
```

Comment:
You will want to check and update the IDL site because for this same section, the anchor link is incorrect.

Reply:
thanks for the heads up!